### DISCUSSION TITLE
New project lead?

### DISCUSSION THREAD

**Discussion by cseickel** [nvim-neo-tree/neo-tree.nvim#1462] [Internal Development]
You may have noticed I've been even more absent than usual from the neo-tree repo lately. I find I just don't have the energy/interest/time to keep maintaining this plugin. I'd like to pass on the role of lead maintainer to someone else.

@pysan3 you've been the most active maintainer recently, are you interested?

**Comment by miversen33** [comment]
If they're up for it, I agree that Pysan would be a fantastic choice

**Comment by pysan3** [comment]
That's a great honor and big thanks to @cseickel and other members for maintaining this project for so long.

Would you still keep an eye on the project tho until merging the big refactor I'm working?

I'm not very familiar with the current codebase and preferably I don't want to devote my time to solve legacy issues, and we might be able to (partially) start from scratch after the migration.

Due to big changes in my life style this April, I won't be able to move quickly but I'm very interesting in leading this project and I think I can resume working on the refactor from mid May.

Is there something I'll need to do in terms of changing the project lead right now?

**Comment by cseickel** [comment]
> That's a great honor and big thanks to @cseickel and other members for maintaining this project for so long.

Thank you for all of your hard work!

> Would you still keep an eye on the project tho until merging the big refactor I'm working?

I will respond to questions from you and I can offer a second set of eyes to PRs for a while. I can't really keep up with the issues though.

> I'm not very familiar with the current codebase and preferably I don't want to devote my time to solve legacy issues, and we might be able to (partially) start from scratch after the migration.

I think you are more familiar than you think. TBH, I think any legacy issues are either:

- Windows path issues
- Performance in specific situations or very large repos
- Extreme edge cases

Of those, only the Windows path issues are worth fixing at this stage, save the rest for v4. I certainly wouldn't bother adding features to v3.

> Due to big changes in my life style this April, I won't be able to move quickly but I'm very interesting in leading this project and I think I can resume working on the refactor from mid May.

It's almost mid-May now, so not really a big deal at all! Either way, responding to issues and merging other people's PRs is just as important as adding new code yourself.

> Is there something I'll need to do in terms of changing the project lead right now?

No, I gave you admin permissions to the repo so you now have as much control as I do. I'll make some sort of announcement this weekend.

THANK YOU!

**Reply by pysan3** [reply]
Thanks a lot for all the announcements and post to reddit.

Just FYI I think I can start working on the 18th (hopefully).
Until then I can't devote my time to read thru all the comments but I'll reply soon.

**Comment by cseickel** [comment]
Hey @pysan3 👋🏼, I hope all is well with you. It looks like you may have stepped away from the project, should we look for a new lead maintainer?

**Reply by miversen33** [reply]
I believe @pysan3  is no longer actively working on this project. As of current the project is floating but has no one actively developing for it. There are quite a few open PRs but I don't feel comfortable reviewing the majority of them as I do not know this code base that well and am not really in the position to give it the proper time and care it deserves to ensure there isn't breakage.

That said, it may be worth tossing another post up on reddit asking for a new lead.

**Reply by cseickel** [reply]
It's a shame, I wish I had the time to work on this but I just don't. I'll put another post on Reddit soon.

**Reply by miversen33** [reply]
There are contributors who are trying to contribute, I think the biggest thing we need is someone to review and make sure the PRs are safe. That unfortunately is not something I think I can handle, I don't know this project that well
